source/macros.tex

```diff
@@ -198,7 +198,7 @@
 \newcommand{\leftshift}[1]{\ensuremath{\, \mathsf{lshift}_#1 \,}}
 
 %% Notes and examples
-\newcommand{\noteintro}[1]{[\,\textit{#1:}\space}
+\newcommand{\noteintro}[1]{[\,\textit{#1}:\space}
```

Comment:
Hm... We could make the spacing slightly smaller, but I think having some spacing is good.